specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_common_options.tsp

```diff
@@ -0,0 +1,30 @@
+import "@typespec/versioning";
+import "./azure_chat_extensions.tsp";
```

Comment:
Are these imports circular references? Is this a problem?

- `extensions_onyourdata_common_options.tsp` imports:
- ----> `./azure_chat_extensions.tsp`, which imports:
- --------> `./extensions_onyourdata_azure_cognitive_search_options.tsp` which imports:
- ------------> `./azure_chat_extensions.tsp`, and also `./extensions_onyourdata_common_options.tsp`